.circleci/config.yml

```diff
@@ -19,9 +19,37 @@ jobs:
       - run: yarn install --ignore-engines
       - run: yarn run test:integration
 
+  deploy-prod:
+    docker:
+      - image: circleci/python:3-node
+    steps:
+      - checkout
+      - run: yarn global add now
+      - run:
+          command: $(yarn global bin)/now  --confirm --prod --token $ZEIT_TOKEN --local-config now.json --scope ccdl
+          no_output_timeout: 1h
+
+  deploy-staging:
+    docker:
+      - image: circleci/python:3-node
+    steps:
+      - checkout
+      - run: yarn global add now
+      - run:
+          command: $(yarn global bin)/now --confirm --prod --token $ZEIT_TOKEN --local-config now.staging.json --scope ccdl
+          no_output_timeout: 1h
```

Comment:
Wait the PR says production but there's both staging and prod here. Why is staging also being added?

Reply:
We deploy both prod/staging with the same command, the only difference is the config file. Also, these are only executed for the `master`/`dev` branches